docs/source/intro/01-install.md

```diff
@@ -48,8 +47,26 @@ There is no `.app` package currently for macOS, but can try Deluge with [Homebre
 
         pip3 install deluge
 
-[develop ppa]: https://launchpad.net/~deluge-team/+archive/ubuntu/develop/
+
+## Alternative Installs
+
+### <i class="icon-python"></i> PyPi
+
+To install from Python PyPi, Deluge requires the following system installed packages:
+
+    sudo apt install python3-pip python3-libtorrent python3-gi python3-gi-cairo gir1.2-gtk-3.0 gir1.2-appindicator3
+
+Install with pip:
+
+    pip3 install deluge
+
+### Developer PPA
```

Comment:
This is actually a stable PPA so shouldn't be called development PPA. It was actually a pain to maintain the original stable PPA and this 'daily' PPA tracks master branch. I think for now just call it Deluge PPA.

Reply:
Does Deluge consider master stable? When I think "stable" I'm only thinking about releases.

Reply:
I guess since there's a develop branch, that is the unstable work.